functional-tests/src/test/java/org/infinispan/online/service/caching/CachingServiceTest.java

```diff
@@ -29,30 +32,60 @@
    @ArquillianResource
    URL restService;
 
+   @Named("caching-service-app-memcached")
+   @ArquillianResource
+   URL memcachedService;
+
    @ArquillianResource
    OpenShiftClient openShiftClient;
 
    ReadinessCheck readinessCheck = new ReadinessCheck();
    HotRodTester hotRodTester = new HotRodTester();
    RESTTester restTester = new RESTTester();
+   MemcachedTester memcachedTester = new MemcachedTester();
 
    @Before
    public void before() {
       readinessCheck.waitUntilAllPodsAreReady(openShiftClient, 30, TimeUnit.SECONDS);
    }
 
    @Test
-   public void should_default_cache_be_accessible_via_hot_rod() throws IOException {
-      hotRodTester.testBasicEndpointCapabilities(hotRodService);
+   @Ignore
+   public void should_not_blow_up_because_of_oom() {
+      hotRodTester.testPutPerformance(hotRodService, 60, TimeUnit.SECONDS);
+   }
+
+   @Test
+   public void restBasicTypesPutGet() throws IOException {
```

Comment:
Use the same naming convention as in other tests.

Reply:
I think the naming convention should be camel case.

Reply:
I think underscore notation (but only in test names) gives you better readability with longer names. Just have a look at the CI job (http://ci.infinispan.org/job/Caching%20Service/job/dataservices-dev/34/testReport/org.infinispan.online.service.caching/CachingServiceTest/). You can read it very easily. Also it works quite nicely when you search for a specific name in Intellij.

So here's my proposal - let's try to use it consequently. If we don't like it, we can change it but I would like all method names to be consistent - either underscore or camel...

//cc @ryanemerson

Reply:
+1 for keeping it consistent that is the most important thing.

Personally, I prefer camelCase and it is the norm in all of the other jdg/infinispan projects. But as @slaskawi says we can change this later.